rushti.py

```diff
@@ -474,25 +475,28 @@ def get_ordered_tasks_and_waits(
 
 
 def execute_process_with_retries(tm1: TM1Service, task: Task, retries: int):
-    attempt = 0
-    while attempt <= retries:
+    for attempt in range(retries + 1):
         try:
-            # process runs and returns either success = True or False
+            # Execute the process and unpack results
             success, status, error_log_file = tm1.processes.execute_with_return(
-                process_name=task.process_name,
+                process_name=task.process_name, 
                 **task.parameters)
+
+            # Handle minor errors for OptimizedTask
+            if isinstance(task, OptimizedTask) and not success and task.succeed_on_minor_errors and status == 'HasMinorErrors':
```

Comment:
Should we respect the `succeed_on_minor_errors` argument also in the NORM run mode?

I

Reply:
Well, the reason I only included it in the Optimized one is because, from both types, is the one you use if you want to have more granular control over the tasks and their outcomes:

> This parameter must OPTIONAL and, IMHO, should only be applicable to OptimizedTasks as they are the ones that you would use when you want to have more precise control on the outcome of the TI processes invoked by RushTI.

That being said there's no technical impediment to include it in the NORM mode too

Reply:
I think we should respect it in the norm mode too for consistency.

People might also find it helpful as the `success` variables drives the 0 or 1 that is returned at the end of the execution when we do `successes=sum(results)`

Reply:
added `succeed_on_minor_errors` to NORM mode on 0843165